### ISSUE TITLE
How to extract undetected object class labels after extracting `y_pred` from `predictions.pth` inference file.

### ISSUE THREAD

**Issue by Eldad27** [facebookresearch/maskrcnn-benchmark#1315]
I training maskrcnn on a custom dataset with two classes. After testing, I get some files ```segm.json```, ```predictions.pth```, ```coco_results.pth``` and ```bbox.json``` in the inference folder. I have successfully extracted ```y_pred``` from ```predictions.pth``` inference file using the code below.

```
import torch
import numpy as np
predictions = torch.load("predictions.pth")
y_preds = []
for box in predictions:
   scores = box.get_field("scores")
   labels = box.get_field("labels")
   mask = box.get_field("mask")
   mask_scores = box.get_field("mask_scores")
   y_preds = np.concatenate((y_preds, labels.numpy()), axis=None)
y_preds = np.array(y_preds, dtype=np.int32)
print(y_pred.shape)
print(y_preds)
```

I get an output for shape and  ```y_pred``` similar to one below. 
```
(17)
[1 1 1 2 1 2 1 1 1 1 2 1 1 1 1 2 1 2]
```

However, this does not add up to ```y_true``` labels (which is 25 classes in total), Meaning some detections (about 8) were missed. How do I extract the 8 classes of objects that weren't detected? How do I know the class(es) that were not detected without running a test on each image? i.e. category labels from ground truth (`y_true`) that are not included in the output.
